### ISSUE TITLE
Display of multiple downloads from AUR overlap

### ISSUE THREAD

**Issue by sclause2412** [Jguer/yay#2282]
### Affected Version

yay v12.1.3 - libalpm v13.0.2

### Describe the bug

When downloading multiple packages from AUR the downloads are done in parallel which is fine. However, the download status is displayed in one line. So the text is flickering around and sometimes shifted and one can read nothing.
This happens only with AUR packages as they are downloaded by yay directly, while official packages are downloaded with pacman which handles the multiple download display correctly.
If the downloads are quite small then you may not see it but if they are quite big (e.g. binaries) it happens regularly.

### Reproduction Steps

1. Install two bigger packages from AUR at the same time (in one command)

Example: yay -S firefox-beta-bin brave-bin

### Expected behavior

If downloads are done parallel they should have their own line like for pacman. As alternative do AUR downloads only one by one.

### Output

% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
7  4836 .161M. 4 M    7  4633 5135k5 1 k    0   0        0   107  8182k3 2 k        0    00 : 000::0409: 0 90 : 000::0003: 0 40 : 000::0406: 0157 8183  7 86.6M    7 6767k    0     0  1401k      0  0:01:03  0:00:04  0:00:59 1401k

**Comment by Jguer** [comment]
Duplicate of https://github.com/Jguer/yay/issues/1620
